src/Run/Executor.cs

```diff
@@ -144,25 +144,22 @@ public static int Main(string[] args)
                 Console.Error.WriteLine("Error: Could not load Json configuration file.");
                 return 1;
             }
-            else
-            {
-                string os = RuntimeInformation.IsOSPlatform(OSPlatform.Windows) ? "windows": "unix";
+            string os = RuntimeInformation.IsOSPlatform(OSPlatform.Windows) ? "windows": "unix";
 
-                jsonSetup.prepareValues(os, executor.SettingParameters, executor.configFilePath);
-                if (executor.DefineParameters(parseArgs, jsonSetup))
+            jsonSetup.prepareValues(os, executor.SettingParameters, executor.configFilePath);
+            if (executor.DefineParameters(parseArgs, jsonSetup))
+            {
+                List<string> paramSelected = new List<string>();
+                foreach (KeyValuePair<string, string> param in executor.CommandParameters[executor.CommandSelectedByUser])
                 {
-                    List<string> paramSelected = new List<string>();
-                    foreach(KeyValuePair<string, string> param in executor.CommandParameters[executor.CommandSelectedByUser])
+                    if (!string.IsNullOrEmpty(param.Value))
                     {
-                        if (!string.IsNullOrEmpty(param.Value))
-                        {
-                            paramSelected.Add(param.Key);
-                        }
+                        paramSelected.Add(param.Key);
                     }
-                    return jsonSetup.ExecuteCommand(executor.CommandSelectedByUser, paramSelected);
                 }
+                return jsonSetup.ExecuteCommand(executor.CommandSelectedByUser, paramSelected);
             }
-            return 0;
+            return 1;
```

Comment:
>return 1; [](start = 12, length = 9)

I would add a comment or something here saying that this means that DefineParameters found an error, or I would add a `Console.error.WriteLine("Error: One or more errors were found on the passed-in parameters")`